target_chains/ethereum/contracts/contracts/pyth/Pyth.sol

```diff
@@ -379,11 +379,11 @@ abstract contract Pyth is
         view
         returns (
             /// @return newOffset The next position in the update data after processing this TWAP update
-            /// @return twapPriceInfo The extracted time-weighted average price information
-            /// @return priceId The unique identifier for this price feed
+            /// @return priceInfos Array of extracted TWAP price information
```

Comment:
maybe change the function name as well? You can call it `extractTwapPriceInfos` or something